vhost-device-vsock/Cargo.toml

```diff
@@ -26,9 +26,8 @@ virtio-queue = "0.12"
 virtio-vsock = "0.6"
 vm-memory = "0.14.1"
 vmm-sys-util = "0.12"
-config = { version = "0.14", default-features = false, features = ["yaml"] }
```

Comment:
After removing it we have the following warning:

```
warning: Patch `config v0.14.0 (https://github.com/mehcode/config-rs?rev=e3c1d0b452639478662a44f15ef6d5b6d969bf9b#e3c1d0b4)` was not used in the crate graph.
Check that the patched package version and available features are compatible
with the dependency requirements. If the patch has a different version from
what is locked in the Cargo.lock file, run `cargo update` to use the new
version. This may also occur with an optional dependency that is not enabled.
```

So we should remove the workaround we did in the main Cargo.toml:
```diff
diff --git a/Cargo.toml b/Cargo.toml
index 58d6deb..bf2d33e 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -12,13 +12,3 @@ members = [
     "vhost-device-template",
     "vhost-device-vsock",
 ]
-
-# In vhost-device-vsock we use the `config` crate which now has the `yaml-rust`
-# dependency that is unmaintained as cargo audit is reporting.
-# See https://github.com/rust-vmm/vhost-device/issues/639
-#
-# They merged the fix, but for now they don't do a release because they don't
-# have a maintainer. To get the CI green, let's do this workaround, but before
-# the next release we have to find a solution.
-[patch.crates-io]
-config = { git = "https://github.com/mehcode/config-rs", rev = "e3c1d0b452639478662a44f15ef6d5b6d969bf9b" }
```